src/stream/operator/mod.rs

```diff
@@ -124,12 +130,16 @@ where
         timer: &'b mut Timer<u64, OP::TimerState, B>,
         channel_strategy: &'c mut ChannelStrategy<OP::OUT>,
         logger: &'d ArconLogger,
+        _name: &'a str,
```

Comment:
Add feature flag above name parameter? in the make_context!() macro you can also add the flag above the borrow to &sel.descriptor etc..